### PR TITLE
Mta 1615 comb assess analysis tabs

### PR DESCRIPTION
**Pull request by HagayVider1**
Replaces PR: https://github.com/windup/windup-documentation/pull/800
Jira issue: MTA-1615 https://issues.redhat.com/browse/MTA-1615
Preview: 
https://deploy-preview-803--windup-documentation.netlify.app/docs/web-console-guide/master/#manage-columns_user-interface-guide

### CODE REVIEW

File: docs/topics/mta-web-manage-columns.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+.Procedure
+
+. In *Migration* view, click *Application inventory*.
+. Click *Manage Collumns* to open a dialog that lists all the available columns.
++
+.Manage columns
+image::mta-web-manage-columns.png[Manage columns]
+
+. Check the box to the left of each columns that you want to appear in the application inventory.
+. Click *Save* to set your selection.
```
Comment (abrugaro) [inline docs/topics/mta-web-manage-columns.adoc:20]: This is not implemented in MTA 7.0.0

**Review by abrugaro — CHANGES_REQUESTED** [review]

**Review by abrugaro — CHANGES_REQUESTED** [review]
See above comment

File: docs/topics/mta-web-edit-application.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+.Prerequisites
+
+* Log in to a server running {ProductShortName}
+
+.Procedure
+
+. In *Migration* view, click *Application Inventory*.
+. Click the row of an application to open a summary of the application's current settings.
++
+// Get updated image from MTA UI
```
Comment (anarnold97) [inline docs/topics/mta-web-edit-application.adoc:20]:
```suggestion

```

File: docs/topics/mta-web-manage-columns.adoc (hunk 1 above)
Comment (anarnold97) [reply docs/topics/mta-web-manage-columns.adoc:20]: @abrugaro - are you ok if we comment out the `include` statement in the `master.adoc`

which would remove this entire page

File: docs/topics/mta-web-edit-application.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+. In *Migration* view, click *Application Inventory*.
+. Click the row of an application to open a summary of the application's current settings.
++
+.View and edit application
+image::mta-web-app-details-01.png[View and edit application]
+
+. To change application settings, click {icon-edit.png} to open the application settings, and review the application settings. See previous procedure for a list of application settings.
+. If you changed any application settings, then click *Save*, otherwise click *Cancel*.
+
+// [Verification]
```
Comment (anarnold97) [inline docs/topics/mta-web-edit-application.adoc:26]: This file does not appear in the preview as it is not included in the `master.adoc`

File: docs/topics/mta-web-manage-columns.adoc (hunk 1 above)
Comment (dymurray) [reply docs/topics/mta-web-manage-columns.adoc:20]: yes please keep this removed.

Comment (abrugaro) [reply docs/topics/mta-web-manage-columns.adoc:20]: @anarnold97 sounds good to me, thanks!

**Review by abrugaro — APPROVED** [review]

File: docs/topics/mta-web-edit-application.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+.Procedure
+
+. In *Migration* view, click *Application Inventory*.
+. Click the row of an application to open a summary of the application's current settings.
++
+.View and edit application
+image::mta-web-app-details-01.png[View and edit application]
+
+. To change application settings, click {icon-edit.png} to open the application settings, and review the application settings. See previous procedure for a list of application settings.
```
Comment (anarnold97) [inline docs/topics/mta-web-edit-application.adoc:23]:
```suggestion
. To change application settings, click `edit` icon {icon-edit} to open the application settings, and review the application settings. See previous procedure for a list of application settings.
```

File: docs/topics/mta-web-manage-columns.adoc (hunk 1 above)
Comment (HagayVider1) [reply docs/topics/mta-web-manage-columns.adoc:20]: Link to page commented out of master.adoc file

File: docs/topics/mta-web-edit-application.adoc [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+.Procedure
+
+. In *Migration* view, click *Application Inventory*.
+. Click the row of an application to open a summary of the application's current settings.
++
+.View and edit application
+image::mta-web-app-details-01.png[View and edit application]
+
+. To change application settings, click `edit` icon {icon-edit} to open the application settings, and review the application settings. See previous procedure for a list of application settings.
```
Comment (shawn-hurley) [inline docs/topics/mta-web-edit-application.adoc:23]: Can we link to the previous procedure here?

**Review by shawn-hurley — APPROVED** [review]
Just a quick thought: everything else looks good to me

File: docs/topics/mta-web-edit-application.adoc (hunk 5 above)
Comment (HagayVider1) [reply docs/topics/mta-web-edit-application.adoc:29]: Added note in beginning of procedure

**Pull request closed without merging**